test/unit/network/libp2p/rpc.test.ts

```diff
@@ -42,6 +44,20 @@ describe("[network] rpc", () => {
       promisify(nodeB.stop.bind(nodeB))(),
     ]);
   });
+
+  //prevents tests from exiting
```

Comment:
Perhaps we can open an accompanying issue for this?

Reply:
@wemeetagain do you know why this would make tests hang. Is this starting some server or opening some connection?

@GregTheGreek Yeah, I was waiting if Cayman has some idea before opening issue

Reply:
What is actually happening in that test? I don't think its actually testing anything..

Reply:
when creating a node, it parses multiaddress, do this checks if all default multiaddresses are correctly formatted.

Reply:
I don't know. As far as I knew, the thing that caused hanging in my experience was `start`ing a node without `stop`ing it.
Here, we're just making a new node, not starting it. No clue :man_shrugging: